cocos/rendering/custom/binary-archive.ts

```diff
@@ -4,7 +4,7 @@ export class BinaryOutputArchive implements OutputArchive {
     constructor () {
         this.capacity = 4096;
         this.buffer = new Uint8Array(this.capacity);
```

Comment:
Would it be better if we changed the name of this place?
= = this.buffer.buffer